modules/ROOT/pages/_partials/api-experience-hub-usage.adoc

```diff
@@ -0,0 +1,20 @@
+= API Experience Hub Usage Reports
+
+include::partial$api-eh-metrics.adoc[tag=intro]
+
+== Access API Experience Hub Usage Reports
+
+include::partial$usage-reports-tasks.adoc[tag=accessUsageReportsNew]
+
+image::api-eh-usage-report.png[Usage Reports interface showing an API Experience Hub usage report that contains a card and table of usage data]
+
+== API Experience Hub Metrics
+
+=== Metrics Cards
```

Comment:
Do these need to be headings since there's not that much content?

Reply:
Those headings aren't being used.